src/clj/clojurians_log/components/parser.clj

```diff
@@ -0,0 +1,23 @@
+(ns clojurians-log.components.parser
```

Comment:
`clojurians-log.components.*` namespaces are for implementing components in the [Component](https://github.com/stuartsierra/component) sense. 

Naming is always hard... Maybe `clojurians-log.slack-messages` or `clojurians-log.slack.messages`?